pkg/builder/builder_test.go

```diff
@@ -406,11 +413,12 @@ func failToReadLockTC(t *testing.T, mockCtrl *gomock.Controller) testCase {
 
 	ctx := context.TODO()
 	buildOpts := builddef.BuildOpts{
-		File:        "zbuild.yml",
-		LockFile:    "zbuild.lock",
-		Stage:       "dev",
-		SessionID:   "<SESSION-ID>",
-		ContextName: "some-context-name",
+		File:          "zbuild.yml",
+		LockFile:      "zbuild.lock",
+		Stage:         "dev",
+		SessionID:     "<SESSION-ID>",
+		SourceContext: "some-context-name",
```

Comment:
it can literally be anything ?

Reply:
It depends :smile: When zbuild is used with docker cli, the context name is always `context`. However when it's used with buildkit, the context name is passed through cli arguments so it could be anything indeed.